packages/components/src/components/hds/code-editor/index.hbs

```diff
@@ -0,0 +1,53 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class={{this.classNames}} ...attributes>
+  {{#if (and this.isSetupComplete (or this.hasToolbarButton (has-block)))}}
+    <div class="hds-code-editor__header">
+      <div class="hds-code-editor__header-content">
+        {{yield
+          (hash
+            Title=(component "hds/code-editor/title")
+            Description=(component "hds/code-editor/description")
+            Button=(component "hds/code-editor/button")
+          )
+        }}
+      </div>
+
+      {{#if this.hasToolbarButton}}
+        <div class="hds-code-editor__header-toolbar">
+          {{#if @hasCopyButton}}
+            <Hds::Copy::Button
+              class="hds-code-editor__button hds-code-editor__copy-button"
+              @isIconOnly={{true}}
+              @size="small"
+              @text="Copy"
+              @textToCopy={{@value}}
+            />
+          {{/if}}
+          {{#if @hasExpandButton}}
+            <Hds::CodeEditor::Button
+              class="hds-code-editor__expand-button"
+              @isIconOnly={{true}}
+              @size="small"
+              @icon="maximize"
+              @text="Fullscreen"
```

Comment:
Haven't done a full review yet, but just noticed this.

The label should be "Toggle full screen view" and then the button needs to have `aria-pressed` set to true or false depending on the state.

Reply:
looping @alex-ju into this because we discussed how it looks in Figma. How we want this to represent should be similar (or I can change to mimic), but this is what it looks like in the properties (status has both maximize and minimize as options):

<img width="806" alt="image" src="https://github.com/user-attachments/assets/92b31002-b278-4cfa-a80e-d85d915aeb4e" />